connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java

```diff
@@ -206,15 +206,18 @@ protected void iteration() {
      */
     private void onCommitCompleted(Throwable error, long seqno, Map<TopicPartition, OffsetAndMetadata> committedOffsets) {
         if (commitSeqno != seqno) {
-            log.debug("{} Got callback for timed out commit: {}, but most recent commit is {}", this, seqno, commitSeqno);
+            log.debug("{} Received out of order callback for commit of generation {}, but most recent generation is {}",
```

Comment:
nit: could we replace "generation" with "sequence"? The term "generation" is already used by the consumer which makes this message a bit confusing. Same for the other log messages below.